### DISCUSSION TITLE
classes/class_audiostreamplaybackpolyphonic

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#272] [User-contributed notes]
# classes/class_audiostreamplaybackpolyphonic

Inherits: AudioStreamPlayback< RefCounted< Object Playback instance for AudioStreamPolyphonic. Description: Playback instance for AudioStreamPolyphonic. After setting the stream property of AudioSt...

https://docs.godotengine.org/en/stable/classes/class_audiostreamplaybackpolyphonic.html

**Comment by BaconEggsRL** [comment]
No get_stream_volume()?

**Comment by Peter-Petersburg** [comment]
The "bus" parameter in play_stream is only used for PLAYBACK_TYPE_SAMPLE. It has no effect for PLAYBACK_TYPE_STREAM.
